src/SkyCoords.jl

```diff
@@ -201,7 +204,7 @@ end
 # matrix (a function of epoch) once and apply it to the entire
 # vector. The compiler doesn't seem to figure out that rotmat(T, S) is
 # a constant for FK5Coords{e} types.
-convert{T<:AbstractSkyCoords,n}(::Type{Array{T,n}}, c::Array{T,n}) = c
+convert{T<:AbstractSkyCoords,n}(::Type{AbstractArray{T,n}}, c::AbstractArray{T,n}) = c
```

Comment:
I think the first argument here should still be `::Type{Array{T,n}}` in order for tests to pass.

Reply:
This did the trick thanks!  But still I don't understand why it works for me...